packages/yourloops/components/dashboard-widgets/medical-files/medical-report.tsx

```diff
@@ -0,0 +1,154 @@
+import TrashCanOutlined from '../../icons/trash-can-outlined'
+import { type MedicalReport } from '../../../lib/medical-files/models/medical-report.model'
+import ListItemButton from '@mui/material/ListItemButton'
+import { getMedicalReportDate } from './medical-report-list.util'
+import { useTeam } from '../../../lib/team'
+import { useUserName } from '../../../lib/custom-hooks/user-name.hook'
+import { VAR_TEXT_BASE_COLOR_DARKER, VAR_TEXT_BASE_COLOR_LIGHT } from '../../theme'
+
+interface MedicalReportItemProps {
+  medicalReport: MedicalReport
+  onClickMedicalReport: (medicalReport: MedicalReport) => void
+  onDeleteMedicalReport: (medicalReport: MedicalReport, teamName: string) => void
+}
+
+const FONT_SIZE_SMALL = '12px'
+const DATE_ITEM_WIDTH = '30%'
+
+const useStyle = makeStyles()((theme) => ({
+  dateItem: {
+    maxWidth: DATE_ITEM_WIDTH,
+    minWidth: DATE_ITEM_WIDTH,
+    width: DATE_ITEM_WIDTH
+  },
+  reportNameItem: {
+    paddingRight: theme.spacing(2)
+  }
+}))
+
+export const MedicalReportItem: FunctionComponent<MedicalReportItemProps> = (props) => {
+  const { medicalReport, onClickMedicalReport, onDeleteMedicalReport } = props
+
+  const { t } = useTranslation('yourloops')
+  const { classes } = useStyle()
+  const { user } = useAuth()
+  const { getTeam } = useTeam()
+  const { getUserName } = useUserName()
+
+  const medicalReportDate = getMedicalReportDate(medicalReport)
+  const medicalReportNumber = medicalReport.number
+  const isUserAuthor = user.id === medicalReport.authorId
+  const authorName = getUserName(medicalReport.authorFirstName, medicalReport.authorLastName, '')
+  const createdBy = authorName.length > 0 ? t('created-by', { name: authorName }) : t('created-by-unknown')
+  const teamName = getTeam(medicalReport.teamId)?.name ?? medicalReport.teamName
+  const deleteMedicalReportLabel = t('delete-medical-report-number', {
+    number: medicalReportNumber,
+    name: teamName
+  })
+
+  return (
+    <ListItem
+      data-testid="medical-report-item"
+      dense
+      divider
+      disablePadding
+      secondaryAction={
+        isUserAuthor &&
+        <Tooltip title={deleteMedicalReportLabel}>
+          <IconButton
+            data-testid="delete-medical-report"
+            edge="end"
+            size="small"
+            aria-label={deleteMedicalReportLabel}
+            onClick={() => {
+              onDeleteMedicalReport(medicalReport, teamName)
+            }}
+          >
+            <TrashCanOutlined />
+          </IconButton>
+        </Tooltip>
+      }
+    >
+      <ListItemButton
+        onClick={() => {
+          onClickMedicalReport(medicalReport)
+        }}
+      >
+        <ListItemIcon>
+          <DescriptionOutlinedIcon />
+        </ListItemIcon>
+        <Box display="flex" justifyContent="space-between" width="100%">
+          <ListItemText
+            className={classes.reportNameItem}
+            primary={
+              <Typography className="bold" fontSize="14px">
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Same remark, here you could use `variant="subtitle2"` instead of setting the `fontSize`